kubernetes/zenko/charts/s3-data/values.yaml

```diff
@@ -4,13 +4,25 @@
 replicaCount: 1
 image:
   repository: zenko/cloudserver
-  tag: 8.0.21
+  tag: 8.0.22
```

Comment:
Can you update the chart `appVersion` as well?

Reply:
Sure